src/Peering/Peering/ChangeLog.md

```diff
@@ -18,10 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Bug Fix in IP Prefix validation to allow 0/30 for DirectConnection
 ## Version 0.1.1
-* Update version of AutoMapper library used
-* Fix case sensitivity bug in `Get-AzPeeringLocation` for `-Kind` parameter
-
+* Updated Automapper 6.2.0 -> 6.2.2 
+* Bug fix in Get-AzPeeringLocation that made Kind case sensative.
```

Comment:
Don't manipulate past entries in the changlog as they directly reflect the release notes published for that version of the module. Only add a new entry under the `Upcoming Release` header.